### PR TITLE
BUG: Adds missing X and Y axes labels in ClassificationReport

### PR DESCRIPTION
**Pull request by pkaf**
ClassificaitonReport had no citing of x- and y-ticklabels, which is now added. 

This PR addresses part of the #1201 issue.

I have made the following changes:

1. Added y-axes labels based on class labels
2. Added x-axes labels based on SCORES_KEYS i.e., ['Precision', 'Recall', 'F1', 'Support'] 

Still to do:
- Pass visual/image tests (with some help)

Screen Shot 2022-01-25 at 4 21 42 pm

### CODE REVIEW

**Comment by pkaf** [comment]
I can push new images into baseimages if I know where are self.binary coming from in https://github.com/pkaf/yellowbrick/blob/develop/tests/test_classifier/test_classification_report.py ?

**Comment by bbengfort** [comment]
@pkaf `self.binary` is a binary classification dataset that's used as a fixture for testing - it's a bit of pytest magic to be sure since it's not imported, but it's coming from `conftest.py`.

The simplest way to regenerate the baseline images is as follows:

1. Remove the baseline images:

```
$ python -m tests.images --clear tests/test_classifier/test_classification_report.py
```

2. Rerun the tests (they will all fail since there are no baseline images):

```
$ pytest tests/test_classifier/test_classification_report.py
```

3. Sync the generated images from the tests with the baseline images:

```
$ python -m tests.images tests/test_classifier/test_classification_report.py
```

Hopefully, when you run the tests again they will pass! Take care to ensure you're only regenerating the images for the classification report tests; if you don't supply the path to the test to our helper module you could end up regenerating all the images. Different operating systems generate images differently, so it might break other tests if that happened.

File: yellowbrick/classifier/classification_report.py [hunk 1]
```diff
@@ -188,6 +190,13 @@ def score(self, X, y):
         y_pred = self.predict(X)
         scores = precision_recall_fscore_support(y, y_pred)
 
+        # Decode the target with the label encoder and get human readable labels
```
Comment (lwgray) [inline yellowbrick/classifier/classification_report.py:193]: @pkaf Are lines 193 to 199 necessary?  Can you explain your logic here?  

@bbengfort Did I miss something here?

**Review by lwgray — CHANGES_REQUESTED** [review]
@pkaf This is wonderful, Thank you so very much for submitting this PR.  We are sorry that it has taken this long to get you a review.  I need to request some changes and provide some clarification.
1. Add fontsize definition to docstrings of Visualizer and quick method
2. Add fontsize as a parameter to the quickmethod
3. Address my comments on Lines 193-199

**Review by lwgray — APPROVED** [review]

**Pull request merged**